pkg/performanceprofile/controller/performanceprofile/components/profile/profile.go

```diff
@@ -56,7 +56,20 @@ func IsPaused(profile *performancev2.PerformanceProfile) bool {
 	return false
 }
 
-// IsRpsEnabled or not RPS mask should be set for all physical net devices
+// IsPyhsicalRpsEnabled checks if RPS mask should be set for all physical net devices
+func IsPyhsicalRpsEnabled(profile *performancev2.PerformanceProfile) bool {
+	if profile.Annotations == nil {
+		return false
+	}
+	isPyhsicalRpsEnabled, ok := profile.Annotations[performancev2.PerformanceProfileEnablePhysicalRpsAnnotation]
+	if ok && isPyhsicalRpsEnabled == "true" {
+		return true
+	}
+
+	return false
+}
+
+// IsPyhsicalRpsEnabled checks if all RPS should be applied
```

Comment:
It should be  IsRpsEnabled

Reply:
this is in the function below. notice we have 2 functions to check the annotations.